.goreleaser.yml

```diff
@@ -0,0 +1,48 @@
+# This is an example .goreleaser.yml file with some sane defaults.
+# Make sure to check the documentation at http://goreleaser.com
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod tidy
+    # you may remove this if you don't need go generate
+    - go generate ./...
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - arm
+      - arm64
+    flags:
+      - -trimpath      
+    ldflags:
+      - -s -w
+dockers:
+  - image_templates:
+    - "costela/docker-etchosts:{{ .Tag }}"
+    - "costela/docker-etchosts:{{ .Major }}"
+    - "costela/docker-etchosts:{{ .Major }}.{{ .Minor }}"
+    - "costela/docker-etchosts:latest"
+archives:
+  - format_overrides:
+      - goos: windows
+        format: zip
+    replacements:
+      darwin: mac    
```

Comment:
since we currently can't support either Windows or OSX (mostly because of the docker-in-VM setup), I think we can also get rid of these, right?
```suggestion
```